docs/content/preview/yugabyte-platform/manage-deployments/edit-universe.md

```diff
@@ -17,8 +17,9 @@ YugabyteDB Anywhere allows you to expand a universe to add more nodes and shrink
 
 ![Edit universe](/images/ee/edit-univ.png)
 
-Using the **Edit Universe** page, you can specify the new intent for the universe. This may include a new configuration of nodes powered by a different instance type.
+Using the **Edit Universe** page, you can specify the new intent for the universe. This may include a new configuration of nodes powered by a different instance type. YugabyteDB Anywhere performs these modifications through the YB-Masters powering the universe. The YB-Masters ensure that the new nodes start hosting the tablet leaders for a set of tablets in such a way that the tablet leader count remains evenly balanced across all the available nodes.
 
-YugabyteDB Anywhere performs these modifications through the YB-Masters powering the universe. The YB-Masters ensure that the new nodes start hosting the tablet leaders for a set of tablets in such a way that the tablet leader count remains evenly balanced across all the available nodes.
+Expansion of universes created with an on-premise cloud provider and secured with third-party certificates obtained from external certification authorities follows a different workflow. For details, see [Expand the universe](../../security/enable-encryption-in-transit#expand-the-universe).
+
+If your universe was created using Kubernetes, you have an option of modifying the Helm chart overrides. To do this, navigate to your universe's **Overiew**, click **Actions > Edit Kubernetes Overrides** to open the **Kubernetes Overrides** dialog and complete it by following instruction provided in [Configure Helm overrides](../../create-deployments/create-universe-multi-zone-kubernetes#configure-helm-overrides).
```

Comment:
The placement of this blurb is confusing. 

Edit universe here refers to the action "Edit Universe" that appears in the UI of a universe. This is different from Edit universe overrides which is a separate action for kubernetes univs. It is probably unlikely the user will discover this blurb here. Can we move it to its own "page" under "manage deployments"?

Also, there was a screenshot attached in the last PR, can we add this screenshot here?

Reply:
Changed